### ISSUE TITLE
Dispensed Drugs Not Displayed on Pharmacy Form

### ISSUE THREAD

**Issue by vogoti** [chap-uzima/IQCare#11]
@chris-githu Hi,for the facilities upgraded from v1 to v2 dispensed drugs are not displayed on the pharmacy form
![DrugsNotDisplayed](https://user-images.githubusercontent.com/31826771/54106138-4030f480-43e6-11e9-8e99-b6ff6e6f5502.png)

**Comment by chris-githu** [comment]
@vogoti @mkaburu 
What's your plan for "Same Point Dispense" sites? Should we update them to 2.0.1?
I would prefer not to maintain them on an old SCM module.
However, we do not have a Same Point Dispense in 2.0.1. They would need to record a prescription and dispense as a PoC.
Thoughts?

**Comment by mkaburu** [comment]
@chris-githu I think all facilities should use SCM as a way of tracking their stocks since they have to account for them to the facility that provided them. Majority of the facilities doing same point dispensing are relatively small and therefore daily workload is relatively low so i think they would manage
My thoughts

**Comment by vogoti** [comment]
@chris-githu I agree with @mkaburu,the challenge i foresee is for sites still doing data reconstruction because if stocks are updated and they dispense them for historical reconstruction data then the stocks might not add up.
Maybe we should set a cutoff date when to implement.
 @carolmaina @irenefaith what are your thoughts?

**Comment by irenefaith** [comment]
@chris-githu  @vogoti  @mkaburu ,I agree no facility should be left using the old SCM . 
@chris-githu  as you come up with a better way of them using SCM I think as of now they should prescribe and dispence like in POC as you suggested.
@vogoti  I don't seem to understand your point on '**Maybe we should set a cutoff date when to implement.'**

**Comment by Muchina-David** [comment]
@irenefaith @chris-githu having all sites dispense like POC would be good. However, with reconstruction historical data is still being entered including pharmacy forms that will reduce the stock quantities. I think what @vogoti means is that a site  would need to complete entering all historical prescriptions e.g. today, physical stock take done same day so that beginning the following day, stocks will be accurate going forward. 
The issue then will be, if they get 2.0.1 now, what stocks will be entered to cater for both reconstruction and future dispensing?

**Comment by carolmaina** [comment]
@vogoti I agree with you with regards to setting a cut off date for implementation as this will ensure accurate stocks in the system.

**Comment by Muchina-David** [comment]
@vogoti @mkaburu Was the matter of using the new Pharmacy Module in sites that have not completed reconstruction (above) concluded?
